4_processing_features/README.md

```diff
@@ -0,0 +1,43 @@
+# 4. Processing Extracted Single Cell Features 
+
+In this module, we present our pipeline for processing outputted `.sqlite` file with single cell features from CellProfiler.
+The processed features are saved into compressed `.csv.gz` for use during statistical analysis.
+
+## Pycytominer
+
+We use [Pycytominer](https://github.com/cytomining/pycytominer) to perform the aggregation, merging, and normalization of the NF1 single cell features.
+
+For more information regarding the functions that we used, please see [the documentation](https://pycytominer.readthedocs.io/en/latest/pycytominer.cyto_utils.html#pycytominer.cyto_utils.cells.SingleCells.merge_single_cells) from the Pycytominer team.
+
+### Normalization
+
+CellProfiler features can display a variety of distributions across cells.
+To facilitate analysis, we standardize all features (z-score) to the same scale.
+
+---
+
+## Step 1: Setup Processing Feature Environment
+
+### Step 1a: Create Environment
+
+Make sure you are in the `4_processing_features` directory before performing the below command.
+
+```sh
+# Run this command in terminal to create the conda environment
+conda env create -f 4.processing_features.yml
+```
+
+## Step 2: Normalize Single Cell Features
+
+### Step 2a: Set Up Paths
```

Comment:
I don't think this step is necessary. This notebook contains only relative paths that should work generically on any machine, right?

Reply:
Well they might put the output from CellProfiler in a different folder. The step is more of an as needed type of deal. I don't think that my pipeline specifically puts the sqlite file in the `Analysis_Output` folder (I think I did this manually). 

I think it is good to leave in their in case the user changes things and doesn't understand why it is not working.